packages/web/hooks/use-feature-flags.ts

```diff
@@ -42,10 +65,11 @@ export const useFeatureFlags = () => {
 
   return {
     ...launchdarklyFlags,
+    ...(process.env.NODE_ENV === "development" ? defaultFlags : {}),
```

Comment:
Thanks for this! This will help external contributors.

We have a dev env for LaunchDarkly that allows us to test FFs in local development. Perhaps we could also check to see if `NEXT_PUBLIC_LAUNCH_DARKLY_CLIENT_SIDE_ID` is env var is populated, and if it is, don't use default flags.

Also, we should probably put this spread operator after `notifications` below so that it get's overridden as well.

Reply:
yea i think we want if `!NEXT_PUBLIC_LAUNCH_DARKLY_CLIENT_SIDE_ID` && `process.env.NODE_ENV === 'development` the use defaultFlags. Otherwise LD wont have control over dev env